### PR TITLE
breaks the old config removal into an include locked on stdline being…

### PR DESCRIPTION
**Pull request by wwaltersp**
… returned from the find

### CODE REVIEW

**Comment by davidpanofsky** [comment]
You'll have to move lines 24-27 also.  While you're at it, please refactor to use the find module since we're using ansible 2 now.

File: tasks/programs_and_groups.yml [hunk 1]
```diff
@@ -26,12 +26,6 @@
...
-    path: "{{supervisor_include_path}}/{{item}}.conf"
-    state: absent
-  with_items: "{{all_supervisor_confs.stdout_lines}}"
-  # this removes the groups.yml config if supervisor_program_groups becomes empty
-  when: item not in (supervisor_programs | map(attribute='name') | list) +
-                    (supervisor_program_groups | length>0) | ternary(['groups'],[])
-  notify: restart supervisor managed programs
+# This feels like a hack around when: variable expansion
+- include: remove_configs.yml
+  when: all_supervisor_confs | length > 0
```
Comment (ghost) [inline tasks/programs_and_groups.yml:33]: Not sure this is a list. I think you might want `all_supervisor_confs.stdout_lines`

**Comment by ghost** [comment]
What is the benefit of this change? Skips the loop when there are no managed programs?

**Comment by wwaltersp** [comment]
The intended was to skip the loop when the find isn't triggered e.g. site.yml. I must have had another change to site.yml because this isn't working as intended today. Closing this PR. Will open a new one with just the find updates cause I have those working.

**Pull request closed without merging**